pages/index.js

```diff
@@ -16,7 +20,15 @@ export default function HomePage({ shoppingItems, onCreateItem }) {
           </StyledTotalItems>
         </ListHeader>
         <Form onCreateItem={onCreateItem} />
-        <ShoppingItemList shoppingItems={shoppingItems} />
+        <ShoppingItemList
+          shoppingItems={shoppingItems}
+          onDeleteItem={onDeleteItem}
+        />
+        {shoppingItems.length === 0 && (
+          <StyledMessage>
+            I feel so empty <span>🥺</span> Add new items with the form above.
```

Comment:
```suggestion
            I feel so empty. <span>🥺</span> Add new items with the form above.
```

If I'm being really pedantic: There's a period missing at the end of the sentence. 😜🙈

Reply:
It is a writing style on social media I would say. A bit more casual, which I intended. Using an emoji instead of punctuation :)